tests/ttnn/unit_tests/operations/test_moreh_norm.py

```diff
@@ -0,0 +1,328 @@
+        pytest.skip("TODO: Check why comp_allclose result is poor on WH_B0.")
+    check_dim(input_shape, dim, keepdim)
+    cpu_x, cpu_dy = make_cpu_tensors(input_shape, dim, keepdim=keepdim)
+
+    expected_y, _ = torch_norm(cpu_x, cpu_dy, p=p, dim=dim, keepdim=keepdim, do_backward=False)
+    actual_y, _ = tt_norm(
+        cpu_x,
+        cpu_dy,
+        p=p,
+        dim=dim,
+        keepdim=keepdim,
+        compute_kernel_options=compute_kernel_options,
+        device=device,
+        do_backward=False,
+    )
+    actual_y = actual_y if keepdim else actual_y.reshape(expected_y.shape)
+
+    pass_y, out_y = comp_allclose(expected_y, actual_y, rtol=rtol, atol=atol)
+    logger.debug(f"output's {out_y}")
+    assert pass_y
+
+
+# def run_moreh_norm_backward(input_shape, p, dim, rtol, atol, device, keepdim=False, compute_kernel_options=None):
+#     check_dim(input_shape, dim, keepdim)
+
+#     cpu_x, cpu_dy = make_cpu_tensors(input_shape, dim, keepdim=keepdim)
+
+#     # expected
+#     _, expected_dx = torch_norm(cpu_x, cpu_dy, p=p, dim=dim, keepdim=keepdim, do_backward=True)
+
+#     # actual
+#     _, actual_dx = tt_norm(
+#         cpu_x,
+#         cpu_dy,
+#         p=p,
+#         dim=dim,
+#         compute_kernel_options=compute_kernel_options,
+#         device=device,
+#         do_backward=True,
+#     )
+
+#     # Check input_grad
+#     pass_dx, out_dx = comp_allclose(expected_dx, actual_dx, rtol=rtol, atol=atol)
+#     logger.debug(f"input_grad's {out_dx}")
+
+#     assert pass_dx
+
+
+@pytest.mark.parametrize("p", [2.0, 2.5, -2.5], ids=["p=2.0", "p=2.5", "p=-2.5"])
+@pytest.mark.parametrize(
+    "dim_rtol_atol",
+    [
+        [[], 0.2, 0.2],
+        [None, 0.2, 0.2],
+        [0, 0.1, 0.1],
+        [1, 0.1, 0.1],
+        [2, 0.1, 0.1],
+        [3, 0.1, 0.1],
+        [[0, 1], 0.1, 0.1],
+        [[0, 1, 2], 0.15, 0.15],
+        [[0, 1, 2, 3], 0.2, 0.2],
+        [[0, 1, 3], 0.15, 0.15],
+        [[0, 2, 3], 0.15, 0.15],
+        [[1, 2], 0.1, 0.1],
+        [[1, 2, 3], 0.15, 0.15],
+        [[1, 3], 0.1, 0.1],
+        [[2, 3], 0.1, 0.1],
+    ],
+    ids=[
+        "global_norm(dim=[])",
+        "global_norm(dim=None)",
+        "N",
+        "C",
+        "H",
+        "W",
+        "NC",
+        "NCH",
+        "NCHW",
+        "NCW",
+        "NHW",
+        "CH",
+        "CHW",
+        "CW",
+        "HW",
+    ],
+)
+@pytest.mark.parametrize(
+    "input_shape",
+    [
+        [TILE_HEIGHT, TILE_WIDTH],
+        [2, 2, 2 * TILE_HEIGHT + 13, 2 * TILE_WIDTH + 13],
+    ],
+)
+@pytest.mark.parametrize("keepdim", [True, False], ids=["keepdim-true", "keepdim-flase"])
+def test_moreh_norm(input_shape, p, dim_rtol_atol, keepdim, device):
+    torch.manual_seed(2024)
+    dim, rtol, atol = dim_rtol_atol
+    run_moreh_norm(input_shape, p, dim, rtol, atol, device, keepdim=keepdim)
+
+
+# @pytest.mark.parametrize("p", [2.0, 2.5, -2.5], ids=["p=2.0", "p=2.5", "p=-2.5"])
+# @pytest.mark.parametrize(
```

Comment:
Why is this commented out?

Reply:
I accidentally commented it out. I have now uncommented it, and the test has passed.